packages/nodes-base/nodes/Files/ExtractFromFile/actions/moveTo.operation.ts

```diff
@@ -121,8 +122,8 @@ export async function execute(
 
 			if (!value) continue;
 
-			const encoding = (options.encoding as string) || 'utf8';
 			const buffer = await this.helpers.getBinaryDataBuffer(itemIndex, binaryPropertyName);
+			const encoding = (options.encoding as string) || (chardet.detect(buffer) as string);
```

Comment:
considering how many other nodes do something similar, my preference would be to add a `this.helpers.detectBinaryEncoding` in core. 
Also, there might be couple of places in core that could also benefit from having auto-detect for encoding.